docs/content/latest/troubleshoot/nodes/trouble-common.md

```diff
@@ -5,49 +5,45 @@ headerTitle: Understanding common error messages
 description: How to understand and recover from common error messages
 menu:
   latest:
-    parent: troubleshoot
+    parent: troubleshoot-nodes
     weight: 800
 isTocNested: true
 showAsideToc: true
 ---
 
 ## Skipping add replicas
 
-When a new node has joined the cluster or an existing node has been removed, you may see errors messages like below:
+When a new node has joined the cluster or an existing node has been removed, you may see errors messages like the following:
```

Comment:
errors messages like the following: --> error messages similar to the following: